lib/Runtime/PlatformAgnostic/Platform/POSIX/UnicodeText.cpp

```diff
@@ -120,30 +121,6 @@ namespace PlatformAgnostic
             }
         }
 
-        uint32 ChangeStringCaseInPlace(CaseFlags caseFlags, char16* stringToChange, uint32 bufferLength)
```

Comment:
>ChangeStringCaseInPlace [](start = 15, length = 23)

is this not needed anymore? I think this was an optimization for ASCII strings.

Reply:
I removed it because we couldn't guarantee in-place conversion for any arbitrary string (any ICU implementation takes a src and dest buffer, and if characters can expand from source to dest, doing conversions in place using the same buffer for both args would produce incorrect results). I suppose I can add this back and fail for non-ASCII?

Reply:
I think I am still against this function because in reality, this is only called when converting JavascriptStrings, which are immutable and can't be changed in place regardless. So, unless we can come up with a case where we are casing a string from native code that _isn't_ a JavascriptString, I think this function just adds more complexity.

Reply:
Sounds good to me. If it regresses perf in a scenario we are tracking we can optimize later.

Reply:
Is it worth submitting an IB run with this change?